services/core-api/tests/factories.py

```diff
@@ -1266,14 +1266,26 @@ class Params:
     project_guid = factory.SelfAttribute('project.project_guid')
     email = factory.Faker('email')
     phone_number = factory.Faker('numerify', text='###-###-####')
-    name = factory.Faker('name')
+    # name = factory.Faker('name')
+    first_name = factory.Faker('first_name')
+    last_name = factory.Faker('last_name')
     is_primary = True
     deleted_ind = False
 
     phone_extension = None
     job_title = None
     company_name = None
 
+    # @factory.post_generation
```

Comment:
got some commented out code left here and for name above!

Reply:
This is removed now.